keps/NNNN-kep-template/README.md

```diff
@@ -432,6 +432,12 @@ The e2e framework does not currently support enabling or disabling feature
 gates. However, unit tests in each component dealing with managing data, created
 with and without the feature, are necessary. At the very least, think about
 conversion tests if API types are being modified.
+
+Additionally, for features that are introducing a new API field, unit tests that
+are exercising the `switch` of feature gate itself (what happens if I disable a
+feature gate after having objects written with the new field) is also critical.
+You can take a look at one potential example of such test in:
+https://github.com/kubernetes/kubernetes/pull/97058/files#diff-7826f7adbc1996a05ab52e3f5f02429e94b68ce6bce0dc534d1be636154fded3R246-R282
```

Comment:
Not sure if we need the example? Or maybe we should add an example to the first comment block too?

Reply:
Multiple people were telling me that they have no idea how to do that - after I was giving them a link it seems that it's actually happening. So I think it's useful.

Regarding the first block, we're much less prescriptive there - I'm fine with updating that, but maybe we can do that in a separate PR?